WalletWasabi/Blockchain/Transactions/TransactionSummary.cs

```diff
@@ -64,4 +42,7 @@ public TransactionSummary(SmartTransaction tx, Money amount, IEnumerable<IInput>
 			return null;
 		}
 	}
+
+	public uint256 GetHash() => Transaction.GetHash();
```

Comment:
What was wrong with `public uint256 TransactionId => Transaction.GetHash();`?

It looks like to me that they are doing the same thing, but now with a different name.

IMHO `summary.TransactionId` is more readable and understandable than `summary.GetHash()`.

_This is not a blocker, just curious._

Reply:
OK, make sense now.